molo/profiles/forms.py

```diff
@@ -66,15 +83,17 @@ class EditProfileForm(forms.ModelForm):
         ),
         required=False
     )
+    mobile_number = PhoneNumberField(required=False)
 
     class Meta:
         model = UserProfile
-        fields = ['alias', 'date_of_birth']
+        fields = ['alias', 'date_of_birth', 'mobile_number']
 
     def clean(self):
         alias = self.cleaned_data.get('alias', None)
         date_of_birth = self.cleaned_data.get('date_of_birth', None)
-        if (alias or date_of_birth):
+        mobile_number = self.cleaned_data.get('mobile_number', None)
+        if (alias or date_of_birth or mobile_number):
             return self.cleaned_data
```

Comment:
Because it's the edit form, They can either update one of the fields or all them